webview/window.py

```diff
@@ -49,7 +49,8 @@ def __init__(self, uid, title, url, html, width, height, x, y, resizable, fullsc
                  background_color, js_api, text_select):
         self.uid = uid
         self.title = make_unicode(title)
-        self.url = None if html else transform_url(url)
+        self.url = None if html else url
+        self.real_url = None
```

Comment:
What are the roles of `self.url` and `self.real_url`? Is the latter supposed to be public?
The `self.url` is a bit of a misnomer, as it does not necessarily reflect the current URL. It is not documented, but should be probably be private.

Reply:
`url` is the thing the user passed in, which might be a URL (`str`), a local path (`str` or `Path`), or a WSGI app (callable).

`real_url` is an actual URL (`str`) that browsers will understand.

Reply:
`window.url` is not used anywhere apart from the initialisation, nor documented, so instead of creating a `window.real_url` how about reusing `window.url` and renaming it to `window._url`?

Reply:
Ok, I can do that.

Reply:
On second thoughts, let's leave it as is. I have renamed window.url to window.original_url.